test/trento/application/integration/telemetry/telemetry_test.exs

```diff
@@ -6,16 +6,17 @@ defmodule Trento.Integration.TelemetryTest do
   import Trento.Factory
 
   alias Trento.Installation
+  alias Trento.Installation.Flavor
   alias Trento.Integration.Telemetry
 
   setup :verify_on_exit!
 
   setup do
-    Application.put_env(:trento, :flavor, "Premium")
+    Application.put_env(:trento, :flavor, Flavor.premium())
```

Comment:
why not just an atom :) ?

Reply:
Strong upvote, we could just set an atom inside the config

Reply:
I switched to a really simpler version, keeping everything as it was.
There is some concerns from @fabriziosestito about the possibility to use atoms.